backend/src/main/java/com/woowacourse/gongseek/article/presentation/dto/ArticleIdResponse.java

```diff
@@ -0,0 +1,16 @@
+package com.woowacourse.gongseek.article.presentation.dto;
+
+import com.woowacourse.gongseek.article.domain.Article;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
```

Comment:
```suggestion
@AllArgsConstructor
@Getter
```
다른 곳처럼 Getter가 아래 생성자가 위에 어떠신가요?